build.xml

```diff
@@ -29,23 +29,23 @@
     <property name="github.ref" value="">
         <!-- string    external    The branch or tag ref that triggered the workflow run. For branches this in the format refs/heads/<branch_name>, and for tags it is refs/tags/<tag_name> -->
     </property>
-    <loadresource property="github.ref-type">
+    <loadresource property="github.ref-type" quiet="true">
```

Comment:
When running ant with `verbose` option, the quiet mode is overriden. So I think it is fine to hardcode "`true`" here, right? (Otherwise, we could have used the value of `$verbose` here too.)

Reply:
Yes, `-v` and `-verbose` respectively seem to override `@quiet`.